dashboard/app/controllers/api/v1/sections_students_controller.rb

```diff
@@ -61,6 +61,21 @@ def bulk_add
     end
   end
 
+  # Remove a student from the section
+  # POST /sections/:section_id/students/:id/remove
+  def remove
+    follower = Follower.where(section: @section.id, student_user_id: @student.id).first
+    return render_404 unless @student && follower
```

Comment:
This is fine, but note we should never be in a situation where only one of these is null (outside a race condition), since `@student` is found [through](https://github.com/code-dot-org/code-dot-org/pull/21305/files#diff-650471959dae2ecd050f72a9fe41739bR3) [@section.students](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/sections/section.rb#L56), and that is mapped through `followers`.